app/build.gradle

```diff
@@ -78,65 +88,67 @@ ext {
     robolectricVersion = '3.3.1'
 }
 dependencies {
-    compile 'com.github.ccrama:JRAW:386e2bcc2c'
-    //JRAW uses Guava, update to 20 to prevent conflicts with Robolectric
-    implementation "com.google.guava:guava:23.0-android"
-    //Updating material-dialogs to any newer version will break all AlertDialogWrapper
-    compile 'com.afollestad.material-dialogs:commons:0.8.6.2'
-    compile 'com.jakewharton:process-phoenix:2.0.0'
-    compile 'org.ligi:snackengage:0.5'
-    compile "com.android.support:design:$supportLibVersion"
-    compile "com.android.support:customtabs:$supportLibVersion"
-    compile "com.android.support:recyclerview-v7:$supportLibVersion"
-    compile "com.android.support:appcompat-v7:$supportLibVersion"
-    compile "com.android.support:support-v13:$supportLibVersion"
-    compile "com.android.support:cardview-v7:$supportLibVersion"
-    compile 'com.android.support:multidex:1.0.2'
-    withGPlayCompile 'com.google.android.gms:play-services-drive:10.2.0'
-    compile 'com.github.rey5137:material:1.2.4'
-    compile 'com.nostra13.universalimageloader:universal-image-loader:1.9.5'
-    compile 'com.davemorrissey.labs:subsampling-scale-image-view:3.6.0'
-    // ↑ minor
-       compile 'com.cocosw:bottomsheet:1.3.0@aar'
-    compile 'com.github.ccrama:AndroidVideoCache:483e44579d'
-    compile 'com.lusfold.androidkeyvaluestore:library:0.1.0@aar'
-    compile 'org.apache.commons:commons-lang3:3.5'
-    compile 'jp.wasabeef:blurry:2.1.0'
-    compile 'com.github.johnkil.android-robototextview:robototextview:4.0.0'
-    // ↑ minor
-    compile 'com.makeramen:roundedimageview:2.2.1'
-    compile 'com.getbase:floatingactionbutton:1.10.1'
-    compile 'com.sothree.slidinguppanel:library:3.3.1'
-    compile 'com.github.suckgamony.RapidDecoder:library:7cdfca47fa'
-    compile 'com.github.dasar:shiftcolorpicker:v0.5'
-    compile 'com.squareup.okhttp3:okhttp:3.7.0'
-    compile 'com.google.code.gson:gson:2.8.0'
-    compile 'com.github.ccrama:TedBottomPicker:496623c9b6'
-    compile 'commons-io:commons-io:2.5'
-    compile('com.github.ozodrukh:CircularReveal:1.3.1@aar') { // ↑ major
```

Comment:
When I switched to implementation, I stopped targeting these `aar` files. This let me drop `transitive=true` so I could include it in the array block.

If there was a reason you were targeting `aar` on this and the other let me know.

Reply:
Honestly only reason was because that library specifically required it, but it seems to be working fine without it after a few days of testing!